package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "5.3.0",
+  "version": "5.3.1",
```

Comment:
Why this ?

Reply:
I am combining two PRs into one. Both would be changing the `package.json`. We can add other commits to the branch before we promote.

Reply:
not ideal as mess up the commit history a bit.

Reply:
i can remove it if you want. i thought we were in a hurry

Reply:
i can't do separate commits on this PR because it is also modifying package.json